q

```diff
@@ -0,0 +1,83 @@
+[1;33mdiff --git i/test/cases/ostree-simplified-installer.sh w/test/cases/ostree-simplified-installer.sh[m
```

Comment:
This file looks as if it was added to the commit by mistake... (https://github.com/osbuild/osbuild-composer/pull/3285/commits/5a4b914dd4a4448b8bc77fbfdc935a29f1c77686)